Fixes the fullnames on TP scroll in the Market

HighCrit opened the pull request:
Non-dungeon TP scrolls have the destination matched by the matcher. So no need to iterate over the entire lore.

Fixes #324

P0keDev reviewed (changes requested):
Add it as a variable in the Variables enum in OverlayConfig and it should be good.

HighCrit committed:
Fixes the fullnames on TP scroll in the Market

Non-dungeon TP scrolls have the destination matched by the matcher. So no need to iterate over the entire lore.

Fixes #324
